readers/api/endpoint_test.go

```diff
@@ -156,8 +158,9 @@ func TestReadAll(t *testing.T) {
 			token:  userToken,
 			status: http.StatusOK,
 			res: pageRes{
-				Total:    uint64(len(messages)),
-				Messages: messages[0:10],
+				PageMetadata: readers.PageMetadata{Offset: 0, Limit: 10},
```

Comment:
The default value of offset is 0 so there is no need to define it. This applies to other cases